.github/workflows/build_tests.yaml

```diff
@@ -45,6 +45,10 @@ jobs:
         install_components: 'beta,gke-gcloud-auth-plugin'
     - name: Verify gcp setup
       run: gcloud info
+    - name: Set Google Cloud CLI properties
```

Comment:
nit to add `Set Google Cloud CLI properties` `to a unused zone to verify --zone arg is passed properly in commands.

Reply:
Do you mean to update the name?